packages/flutter_tools/test/web.shard/output_web_test.dart

```diff
@@ -77,5 +77,5 @@ void main() {
     await start();
     await sendEvent(<String, Object>{'type': 'DevtoolsEvent'});
     await warning;
-  });
+  }, skip: 'https://github.com/dart-lang/webdev/issues/1562');
```

Comment:
To get analysis to pass, you will need:

```suggestion
  }, skip: true); // https://github.com/dart-lang/webdev/issues/1562
```

Per https://github.com/flutter/flutter/wiki/Tree-hygiene#skipped-tests

Reply:
Fixed in next commit, thanks!